### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by InventoCasa** [schlingensiepen/TelegramBotClient#6]
Unfortunately I can't get your library to work. I can connect to my wifi without problems, but I can neither receive nor send messages to my telegram bot.

I tried your example ESP_EchoBotKeyboard.ino without changes (except the telegram bot token and my wifi credentials), but no messages are received.

Could you please test it for yourself?

**Comment by jarcher12** [comment]
I have to agree with with you its not working. i have tried with JSON ver 5.13.5 and 5.13.3 neither does anything, it logs onto WiFi tells me WiFi strength and ip adress then absolutely nothing no matter how many times i try to send to the bot

**Comment by sanderkob** [comment]
If you are using esp8266 core 2.5.0 you should add the line
   ssPollClient.setInsecure();
to void setup();

See e.g. this duscussion:
https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/issues/104

**Comment by SeppPenner** [comment]
@sanderkob Yep, client.setInsecure(); solves this as described in the link :)
